src/biophysics.jl

```diff
@@ -260,77 +258,96 @@ function convection(Body, area, T_air, T_surf, vel, P_atmos, elev, fluid)
+    # Bird, Stewart & Lightfoot, 1960. Transport Phenomena. Wiley.
+    Sh_free = Nu_free * (Sc / Pr)^(1 / 3) # Sherwood number, free
+    # calculating the mass transfer coefficient from the Sherwood number
+    Hd_free = Sh_free * dif_vpr / D # mass transfer coefficient, free
+    Q_free = Hc_free * A_conv * (T_surf - T_air) # free convective heat loss at surface
+
+    # forced convection
     Nu = get_Nusselt_forced(Body.shape, Re)
-    #C     FORCED CONVECTION FOR ANIMAL
-    Hc_forc = Nu * k_fluid / D # HEAT TRANFER COEFFICIENT
-    Sh_forc = Nu * (Sc / Pr)^(1 / 3) # SHERWOOD NUMBER
-    Hd_forc = Sh_forc * dif_vpr / D # MASS TRANSFER COEFFICIENT
-    #C     USING BIRD, STEWART, & LIGHTFOOT'S MIXED CONVECTION FORMULA (P. 445, TRANSPORT PHENOMENA, 2002)
+    # forced convection for object
+    Hc_forc = Nu * k_fluid / D # heat transfer coefficient, forced
+    Sh_forc = Nu * (Sc / Pr)^(1 / 3) # Sherwood number, forced
+    Hd_forc = Sh_forc * dif_vpr / D # mass transfer coefficient
+    Q_forc = Hd_forc * A_conv * (T_surf - T_air) # forced convective heat transfer
+
+    # combined free and forced convection
+    # using Bird, Stewart & Lightfoot's mixed convection formula (p. 445, Transport Phenomena, 2002)
     Nu_comb = (Nu_free^3 + Nu^3)^(1 / 3)
-    Hc = Nu_comb * (k_fluid / D) # MIXED CONVECTION HEAT TRANSFER
-    Q_conv = Hc * A_conv * (T_surf - T_air) # TOTAL CONVECTIVE TRANSFER
-    #C     CALCULATING THE SHERWOOD NUMBERs FROM THE COLBURN ANALOGY
-    #C     (BIRD, STEWART & LIGHTFOOT, 1960. TRANSPORT PHENOMENA. WILEY.
-    Sh = Nu_comb * (Sc / Pr)^(1 / 3) # SHERWOOD NUMBER
-    Hd = Sh * dif_vpr / D # MASS TRANSFER COEFFICIENT
-    (Q_conv=Q_conv, Hc=Hc, Hd=Hd, Sh=Sh, Q_free=Q_free, Hc_free=Hc_free, Hc_forc=Hc_forc, Sh_free=Sh_free, Sh_forc=Sh_forc, Hd_free=Hd_free, Hd_forc=Hd_forc)
+    Hc = Nu_comb * (k_fluid / D) # mixed convection heat transfer
+    Q_conv = Hc * A_conv * (T_surf - T_air) # total convective heat loss
+    Sh = Nu_comb * (Sc / Pr)^(1 / 3) # Sherwood number, combined
+    Hd = Sh * dif_vpr / D # mass transfer coefficient, combined
+    (Q_conv=Q_conv, Hc=Hc, Hd=Hd, Sh=Sh, Q_free=Q_free, Q_forc=Q_forc, Hc_free=Hc_free, Hc_forc=Hc_forc, Sh_free=Sh_free, Sh_forc=Sh_forc, Hd_free=Hd_free, Hd_forc=Hd_forc)
 end
 
-function conduction(A, L, T_surf, T_sub, k_sub)
-    A * (k_sub / L) * (T_surf - T_sub)
+function conduction(A_cond = 0.001325006m^2,
+    L=0.025m,
+    T_surf=K(25°C),
+    T_sub=K(10°C),
+    k_sub=0.1W/m/K)
+    A_cond * (k_sub / L) * (T_surf - T_sub)
 end
 
-function solar(α_org_dorsal, α_org_ventral, A_sil, A_up, A_down, F_sub, F_sky, α_sub, Q_dir, Q_dif)
+function solar(α_org_dorsal=0.85,
+    α_org_ventral=0.85,
+    A_sil=0.004718043m^2,
+    A_tot=0.01325006m^2,
+    A_cond=0.001325006m^2,
+    F_sub=0.4,
+    F_sky=0.4,
+    α_sub=0.8,
+    Q_sol=1000W / m^2,
+    Q_dir=964.177772475912W / m^2,
+    Q_dif=100W / m^2)
+
     Q_direct = α_org_dorsal * A_sil * Q_dir
```

Comment:
Q_dir and Q_direct are not being used, right?

Reply:
Q_direct is added to Q_sol_sub + Q_sol_sky at the end ...

Reply:
Oh my... You are right!